Automated build of Sphinx docs fails

Automated build of Sphinx docs fails because sphinx_rtd_theme is not a hard dependency as of Sphinx version 1.4.0. As a result, the Github Pages generation also fails.

Full error logs follow:
```
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v1.4.1
making output directory...
loading pickled environment... not yet created

Theme error:
sphinx_rtd_theme is no longer a hard dependency since version 1.4.0. Please install it manually.
make: *** [html] Error 1
Makefile:55: recipe for target 'html' failed
```

--- a/docs/Dockerfile
+++ b/docs/Dockerfile
@@ -7,7 +7,9 @@ RUN        apt-get update && apt-get install -y --no-install-recommends \
         && rm -rf /var/lib/apt/lists/*
 
 RUN        easy_install pip
-RUN        pip install sphinx
+RUN        pip install \
+           sphinx \
+           sphinx_rtd_theme
 
 VOLUME     /opt/B2HANDLE/docs
 

